File: Examples/Features/Workbook/CreateNewWorkbook.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace GrapeCity.Documents.Excel.Examples.Features.Workbook
{
    public class CreateNewWorkbook : ExampleBase
    {
        public override void Execute(GrapeCity.Documents.Excel.Workbook workbook)
        {
            //Create empty workbook, contains one worksheet default.
            GrapeCity.Documents.Excel.Workbook workbookNew = new GrapeCity.Documents.Excel.Workbook();
        }

        public override bool ShowViewer
        {
            get
            {
                return false;
            }
        }
    }
}

File: Examples.VB/Features/Workbook/CreateNewWorkbook.vb
Namespace Features.Workbook
    Public Class CreateNewWorkbook
        Inherits ExampleBase
        Public Overrides Sub Execute(workbook As Excel.Workbook)
            'Create empty workbook, contains one worksheet default.
            Dim workbookNew As New Excel.Workbook
        End Sub
        Public Overrides ReadOnly Property ShowViewer As Boolean
            Get
                Return False
            End Get
        End Property
    End Class
End Namespace
